### ISSUE TITLE
Test case: Add New Attribute

### ISSUE THREAD

**Issue by dmitry-vrachev** [Shcherbich/QLdapEditor#28]
#### Story description:
Attributes of single existing entity of DIT can be edited. Editing capabilities for the attributes' names and types are restricted by selected scheme, for the attributes' values - by attribute type.

### Use case: add new attribute

**Precondition**:
* Some object is selected in DIT.

---

**User action**: User right-clicks anywhere in the attributes grid, selects "New attribute".

**System reaction**: New attribute string is added below existing strings.

**Exception**: No.

---

**User action**: User opens a dropdown list in the new attribute string.

**System reaction**: Dropdown list contains all possible attribute names read from scheme for the object of current objectClass.

**Exception**: No.

---

**User action**: User selects an attribute name from dropdown list.

**System reaction**: Type is auto-selected for the selected name according to scheme. Selected name cannot be changed anymore. The attribute is displayed with bold until saving.

**Exception**: No.

---

**User action**: User edits value of new attribute.

**System reaction**: See Use case: edit value of existing attribute.

**Exception**: No.

---

**User action**: User saves new attribute

**System reaction**: See Use case: save changes.

**Exception**: No.

**Comment by d-markevich** [comment]
> Dropdown list contains all possible attribute names read from scheme for the object of current objectClass.

Если писать сложно то это должно быть так - все доступные для данного объекта атрибуты, доступные атрибуты это атрибуты определенные классами, которые применены к объекту, за исключением атрибутов с мультиплисти 1, если они уже определены.

Схема определяет все классы, запись наследует классы, может иметь атрибуты этих классов.

**Comment by d-markevich** [comment]
Т.е. точно не current objectClass, не единственное число, и не "объект класса", а "классы примененные к объекту", и не read from scheme а определенные для этих классов в схеме.

**Comment by dmitry-vrachev** [comment]
Да, сейчас не финальный вариант, я пока что старый большой тест-кейс разбиваю как есть. В добавлении атрибутов мы делаем изменения, я буду сюда вносить изменения. Чтобы такие случаи были видны - я ставлю тэг Draft, документы с таким тэгом смотреть пока не надо.

**Comment by d-markevich** [comment]
@dmitry-vrachev  А что оно у нас в In progress висит?
